update accessibility shortcuts (#88329)
Azure/azure-content: articles/machine-learning/designer-accessibility.md

## Navigate the pipeline graph

> The pipeline graph is organized as a nested list. The outer list is a component list, which describes all the components in the pipeline graph. The inner list is a connection list, which describes input/output ports of a specific component connection details. Use below keyboard action to navigate a pipeline graph when focus moved to it

Reviewer: ```suggestion
The pipeline graph is organized as a nested list. The outer list is a component list, which describes all the components in the pipeline graph. The inner list is a connection list, which describes input/output ports of a specific component connection details. The following keyboard actions help you to navigate a pipeline graph. 
```

### Edit a component

#### To connect a component to another component:

> 1. 

Reviewer: @xiaoharper looks like step 1 is blank?
